lib/eco_service_web/live/eco_service_live/community_list.html.heex

```diff
@@ -9,13 +9,19 @@
       <.link href={~p"/community/#{community.id}/edit"} > <span class="text-blue-500">Edit</span> </.link>    
   </:col>
 </.table>
- 
-<%= if @offset != 0 do %>  
-<.link phx-click="prev" ><p class="text-blue-500">Prev</p></.link>
-<% end%>
-<%= if @all_records - @offset > @limit do %>
-<.link phx-click="next" ><p class="text-blue-500">Next</p></.link>
-<% end %>
+
+<div class="flex flex-cols gap-6"> 
+  <%= if @offset != 0 do %>  
+  <.link phx-click="first" ><p class="text-blue-500">First</p></.link>
+  <.link phx-click="prev" ><p class="text-blue-500">Prev</p></.link>
+  <% end%>
+
+  <%= if @count_of_all_records - @offset > @limit do %>
+    <.link phx-click="next" ><p class="text-blue-500">Next</p></.link>
+    <.link phx-click="last" ><p class="text-blue-500">Last</p></.link>
+  <% end %>
+</div>  
```

Comment:
Your logic above doesn't seem very correct to me. It seems a bit problematic. Look at this [tutorial](https://online.pragmaticstudio.com/courses/liveview-starter-1/steps/19) to get an idea of how it is implemented in real world.

Reply:
Is the below tutorial that you are referring to? 
https://online.pragmaticstudio.com/courses/liveview-starter/steps/1
But it is having a lot of video, Can you please mention which video are you referring to?

Reply:
I just checked it. Pragamatic studio's older tutorial used to have pagination but the newer doesn't 😞 

I'll look to organize a meeting to teach you and whoever interested in pagination 😉

Reply:
okay